provision-contest/ansible/roles/judgedaemon/tasks/main.yml

```diff
@@ -34,7 +34,7 @@
     dest: /tmp/dj_ansible/
 
 - name: Create chroot
-  shell: "set -o pipefail && {{ DJ_DIR }}/misc-tools/dj_make_chroot -y -i openjdk-17-jdk-headless -l \"$(ls /tmp/dj_ansible/install-chroot/*.deb 2>/dev/null | tr '\n' ',')\" -s \"$(ls /tmp/dj_ansible/chroot-list/*.list 2>/dev/null | tr '\n' ',')\" 2>&1 | tee /tmp/dj_make_chroot.log; grep '^Done building chroot in' /tmp/dj_make_chroot.log"
+  shell: "set -o pipefail && {{ DJ_DIR }}/misc-tools/dj_make_chroot -y -H -i icpc-kotlinc,openjdk-17-jdk-headless -l \"$(ls /tmp/dj_ansible/install-chroot/*.deb 2>/dev/null | tr '\n' ',')\" -s \"$(ls /tmp/dj_ansible/chroot-list/*.list 2>/dev/null | tr '\n' ',')\" 2>&1 | tee /tmp/dj_make_chroot.log; grep '^Done building chroot in' /tmp/dj_make_chroot.log"
```

Comment:
Nit: Is it possible to break this shell command over multiple lines to increase readability?

Reply:
I'll see what is possible